applicator.json

```diff
@@ -0,0 +1,61 @@
+{
+    "$schema": "http://json-schema.org/draft-08/schema#",
+    "$id": "https://json-schema.org/draft-08/applicator",
+    "$recursiveAnchor": true,
+    "$vocabulary": {
+        "https://json-schema.org/draft-08/vocabularies/core": true,
+        "https://json-schema.org/draft-08/vocabularies/applicator": true
+    },
+
+    "title": "Applicator vocabulary meta-schema",
+    "$defs": {
+        "schemaArray": {
+            "type": "array",
+            "minItems": 1,
+            "items": { "$recursiveRef": "#" }
+        }
+    },
+    "properties": {
+        "additionalItems": { "$recursiveRef": "#" },
+        "unevaluatedItems": { "$recursiveRef": "#" },
+        "items": {
+            "anyOf": [
+                { "$recursiveRef": "#" },
+                { "$ref": "#/$defs/schemaArray" }
+            ]
+        },
+        "contains": { "$recursiveRef": "#" },
+        "additionalProperties": { "$recursiveRef": "#" },
+        "unevaluatedProperties": {
+            "type": "object",
+            "additionalProperties": {
+                "$recursiveRef": "#"
+            }
+        },
+        "properties": {
+            "type": "object",
+            "additionalProperties": { "$recursiveRef": "#" },
+            "default": {}
+        },
+        "patternProperties": {
+            "type": "object",
+            "additionalProperties": { "$recursiveRef": "#" },
+            "propertyNames": { "format": "regex" },
+            "default": {}
+        },
+        "dependentSchemas": {
```

Comment:
Do we also need `dependentRequired`?

Reply:
@gregsdennis yes, but I'll do a separate PR and fix up the meta-schemas properly, as they have a variety of issues unrelated to this actual PR.

Reply:
@gregsdennis actually, no, apologies for the confusion.  `dependentSchemas` is an applicator, and belongs in this meta-schema.  `dependentRequired` is an assertion, and belongs in the assertion vocabulary meta-schema, which is not part of this PR (I'll do a vocabulary-related PR on the validation and hyper-schema specs after this one is merged).